proto/api/v1/api.proto

```diff
@@ -481,6 +481,12 @@ service ConnectorService {
       }
     };
   };
+  rpc ValidateConnector(ValidateConnectorRequest) returns (ValidateConnectorResponse) {
+    option (google.api.http) = {
```

Comment:
We need to add a `grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation` option and specify that the response can contain the http status 400.